docs/advanced/admission.md

```diff
@@ -9,3 +9,7 @@ title: Control resources on the edge with Admission
 toc: true
 type: docs
 ---
+
+:::info
+Work in Progress
+:::
```

Comment:
@fisherxu I've added WIP Info sections for blank pages to prevent users from getting confused